src/app/workflow-editor/object-inspector/step-inspector/tabs/step-tab-step.component.ts

```diff
@@ -225,4 +233,12 @@ export class WorkflowStepInspectorTabStep extends DirectiveBase implements OnIni
         hints.model = this.step;
         hints.readonly = this.readonly;
     }
+
+    setDisabledState(isDisabled: boolean) {
+        const excludedControls = ["scatter", "scatterMethod"];
```

Comment:
What is the point of excluded controls in this case?
You can easily remove disableScatter method and put everything in setDisabledState method.